client/driver/lxc_test.go

```diff
@@ -200,3 +201,98 @@ func TestLxcDriver_Open_Wait(t *testing.T) {
 func lxcPresent(t *testing.T) bool {
 	return lxc.Version() != ""
 }
+
+func TestLxcDriver_Volumes_ConfigValidation(t *testing.T) {
+	if !testutil.IsTravis() {
+		t.Parallel()
+	}
+	if !lxcPresent(t) {
+		t.Skip("lxc not present")
+	}
+	ctestutil.RequireRoot(t)
+
+	brokenVolumeConfigs := [][]string{
+		[]string{
+			"foo:/var",
+		},
+		[]string{
+			":",
+		},
+		[]string{
+			"abc:",
+		},
+		[]string{
+			":def",
+		},
+		[]string{
```

Comment:
Tests are failing because our linter prefers the simplified version of this code. You can fix it with either `gofmt -s lxc_test.go` or just by using this:

```go
        brokenVolumeConfigs := [][]string{
                {
                        "foo:/var",
                },
                {
                        ":",
                },
                {
                        "abc:",
                },
                {
                        ":def",
                },
                {
                        "abc:def:ghi",
                },
        }
```

A bit of a silly code style quibble, but it keeps our code consistent and linters happy!